Duplicati/CommandLine/Commands.cs

```diff
@@ -234,14 +235,86 @@ public static int Affected(TextWriter outwriter, Action<Duplicati.Library.Main.C
             }
         }
 
-        public static List<string> PrefixArgsWithAsterisk(IList<string> argList)
+        /// <summary>
+        /// For bare file names with no wildcards, replace argument with
+        /// the equivalent of prefixing with "*/" so we search all folders.
+        /// </summary>
+        public static IEnumerable<string> PrefixArgsWithAsterisk(IEnumerable<string> argList) =>
+            argList.Select(PrefixArgWithAsterisk);
+
+        /// <summary>
+        /// For bare file names with no wildcards, return argument with
+        /// the equivalent of prefixing with "*/" so we search all folders.
+        /// </summary>
+        private static string PrefixArgWithAsterisk(string arg)
         {
-            // Prefix all filenames with "*/" so we search all folders
-            var specialArgChars = new[] { '*', '?', Path.DirectorySeparatorChar, Path.AltDirectorySeparatorChar };
-            return argList.Select(arg => arg.IndexOfAny(specialArgChars) < 0
-                 && !arg.StartsWith("[", StringComparison.Ordinal) ? "*" + Util.DirectorySeparatorString + arg : arg).ToList();
+            var containsSeparators = ContainsDirectorySeparators(arg);
+            var containsWildcards = ContainsWildcards(arg);
+
+            if (!containsSeparators && arg.StartsWith("@", StringComparison.Ordinal))
+            {
+                // Convert to Regexp filter and prefix with ".*/"
+                return $"[.*{Regex.Escape(Util.DirectorySeparatorString + arg.Substring(1))}]";

```

Comment:
Instead of `Regex.Escape` here, can we use the `Utility.ConvertLiteralToRegExp` helper method to isolate the conversion logic in one place?

Reply:
Fixed in latest commit.